File: QuestaoUm/CurtoCircuito.py
msg = 'x = {}'
print('Bitwise OU');
x = 0;
for i in range(5):
 x += 1;
 if((1==1) | (x<3)):
  print(msg.format(x));
print('\nOU');
x = 0;
for i in range(5):
 x += 1;
 if((1==1) or (x<3)):
  print(msg.format(x));

File: QuestaoUm/CurtoCircuito.java
public class CurtoCircuito{
 public static void main (String[]args){
 System.out.println("Bitwise OU");
 int x = 0;
  for(int i = 0; i < 5; i++){
   if((1==1) | (x++>3))
    System.out.println("x = " + x);
  }
  System.out.println("\nOU");
  x = 0;
  for(int i = 0; i < 5; i++){
   if((1==1) || (x++>3))
    System.out.println("x = " + x);
  }
 }
}

